pkg/test/nodes.go

```diff
@@ -67,15 +68,23 @@ func Node(overrides ...NodeOptions) *v1.Node {
 	}
 }
 
+// this check is done for tests that assume the nodeclaim is past registration but will not go through reconciliation
```

Comment:
Can we specify that it won't be reconclied by the NodeClaim Lifecycle Controller?